src/vm/docs/mod.rs

```diff
@@ -241,6 +241,29 @@ Note: This function is only available starting with Stacks 2.1.",
 "#,
 };
 
+const IS_STANDARD_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(is-standard principal1)",
+    description:
+"Tests whether `principal1` matches the current network type, and therefore
+represents a principal that can spend tokens on the current network type. That is, the network is either of
+type `mainnet`, or `testnet`. And, `principal1` is either suited for,
+or able to spend tokens on, the mainnet or a testnet.  `SPxxxx` and
+`SMxxxx` addresses can only spend tokens on mainnet, whereas `STxxxx` and `SNxxxx` addresses can only
+spend tokens on a testnet. Other than these four forms of address, no other forms of addresses are valid.
```

Comment:
All c32check address forms are valid in that they can receive tokens, but only these four c32check forms can spend them.